UCSRest/src/main/java/it/cnr/iit/ucsrest/rest/UCSRestController.java

```diff
@@ -86,7 +97,7 @@ public Boolean sendMessage( @RequestBody( ) StartAccessMessage message ) {
     // TODO UCS-34 NOSONAR
     public Boolean sendMessage( @RequestBody( ) EndAccessMessage message ) {
         Reject.ifNull( message );
-        log.log( Level.INFO, "Endaccess received {0}", System.currentTimeMillis() );
+        log.log( Level.INFO, "EndAccess received {0}", System.currentTimeMillis() );
```

Comment:
well spotted :-)

Reply:
thanks again ;)
actually we had many logs that use any scramble of *AccEsS and this makes everything the same.